lib/http-context.js

```diff
@@ -192,27 +191,45 @@ function coerce(str) {
 // coerce every string in the given object / array
 function coerceAll(obj) {
   var type = Array.isArray(obj) ? 'array' : typeof obj;
+  var i;
+  var n;
 
   switch (type) {
     case 'string':
       return coerce(obj);
     case 'object':
       if (obj) {
-        Object.keys(obj).forEach(function(key) {
+        var props = Object.keys(obj);
```

Comment:
If obj is prototype-less or if you don't care that prototype properties get iterated over, using `for (var key in obj)` is usually faster.